packages/browser/src/client/tester/locators/index.ts

```diff
@@ -185,6 +185,15 @@ export abstract class Locator {
     return this.elements().map(element => this.elementLocator(element))
   }
 
+  public nth(index: number): Locator {
+    const element = this.elements()[index]
```

Comment:
Playwright supports `nth=` selector: https://github.com/microsoft/playwright/blob/3ec8ee7a9b9ee682f63ff155a36ea868951d8ae5/packages/playwright-core/src/client/locator.ts#L211

Reply:
So it does. I reimplemented `nth()` to call the protected `locator` ... I admit that I am ignorant of the Playwright locator syntax or its role in the `@vitest/browser`, but I copied the `>> nth=` notation from Playwright and the blog tests still pass, so it appears successful. Let me know if that's what you had in mind.

The exception message is still a bit misleading i.e. it does not mention the `nth` chain when translating back to javascript:

```
Cannot find element with locator: locator('body').getByRole('listitem')
```

It looks like Ivya might be responsible for translation. Is it worth looking into this for clearer error reporting?

Reply:
You can look into it if you want if you think it's important. It's also just a fork of playwright's locators implementation